cmd/manager/main.go

```diff
@@ -76,6 +68,24 @@ func main() {
 
 	pflag.Parse()
 
+	// add version subcommand to query the operator version
+	if len(os.Args) > 1 {
+		switch os.Args[1] {
+		case "version":
+			fmt.Printf("%s version: %q, go version: %q\n", os.Args[0], version.Get(),
+				version.GoVersion)
+			os.Exit(0)
+		default:
+			fg := strings.Split(os.Args[1], "=")
+			arg := strings.Replace(fg[0], "--", "", -1)
```

Comment:
And are you sure that they're are coming as pflags? I thought you can also pass flags with `-` instead of `--`, if they're POSIX compliant, only then they would need `--`, is what I thought.

Reply:
`-` is being used for shorthands and `--` for long, `Lookup` command needs `--` to be removed to match the flag, not the case for shorthands. Thats why its able to differentiate between shorthands and long flags in this logic.

Reply:
There is a difference. Usually `-shorthand` means, you won't give the full name.  For example `--log-level` can be made `-v`.The fact that you are `pflag` library makes me think the flags are POSIX compliant.